src/eth_rpc/servers/net_rpc.rs

```diff
@@ -23,8 +22,12 @@ impl<P: Provider + Send + Sync> NetRpc<P> {
 
 #[async_trait]
 impl<P: Provider + Send + Sync + 'static> NetApiServer for NetRpc<P> {
-    fn version(&self) -> Result<U64> {
-        Ok(CHAIN_ID.into())
+    async fn version(&self) -> Result<U64> {
```

Comment:
can you add a small rustdoc comment here saying net_version method returns the network ID

Reply:
done